PxCs/Data/Vob/PxVobData.cs

```diff
@@ -18,7 +18,7 @@ public class PxVobData
         public PxVobData[] childVobs = default!;
 
         public Vector3 position;
-        public PxMatrix3x3Data rotation = default!;
+        public PxMatrix3x3Data? rotation;
```

Comment:
Other files also worked with this  = default! for arrais. Therefor let's have it the same way, please.

```suggestion
        public PxMatrix3x3Data rotation = default!;
```